src/sentry/search/snuba/executors.py

```diff
@@ -1524,9 +1548,15 @@ def query(
                 request, referrer="search.snuba.group_attributes_search.hits"
             )["data"][0]["count"]
 
+        def get_sort_val(row):
+            if sort_by == "new":
+                return row["g.group_first_seen"]
```

Comment:
Is there a way we can alias this as score rather than needing to special case?

Reply:
@wedamija is there a way to alias a column? Also, wouldn't doing that cause a conflict on the name since score is already aliased?

Reply:
Oh nm, I misread what score was doing. I think that we should figure out a way to store the name we expect in the results inthe same place that we define the sorts, so that we can just do `row[SORT_DEFS[sort].col_name]` here (not this exact implementation but that idea)

Reply:
@wedamija fixed